archive-index.html

```diff
@@ -0,0 +1,350 @@
+                </button>
+                <a class="navbar-brand" href="#">PDG Poetry</a>
+            </div>
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav">
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Services</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+    <!-- Page Content -->
+    <div class="container">
+
+        <div class="row">
+
+            <!-- Blog Entries Column -->
+            <div class="col-md-8">
+
+                <h1 class="page-header">
+                    Poetry Archive
+                    <small>A collection of poems</small>
+                </h1>
+              
+              <ul id="poems-list">
+                
+              </ul>
+
+                <!-- First Poem -->
+                <h2>Caged Bird</h2>
+                <p class="lead">
+                    by Maya Angelou
+                </p>
+                <p><span class="glyphicon glyphicon-book"></span> Published: 1983</p>
+                <hr>
+                <img class="img-responsive" src="http://placehold.it/900x300" alt="">
+                <hr>
+                <p>
+                  “The caged bird sings <br>
+                  with a fearful trill <br>
+                  of things unknown <br>
+                  but longed for still”
+                </p>
+                <a class="btn btn-primary" href="#">Read More <span class="glyphicon glyphicon-chevron-right"></span></a>
+                <a class="btn btn-primary" href="/media/Maya-Angelou-Caged-Bird.mp3">Listen <span class="glyphicon glyphicon-music"></span></a>
+                <hr>
+
+                <!-- Second Poem -->
+                <h2>On the Pulse of Morning</h2>
+                <p class="lead">PoemPoem
+                    by Maya Angelou
+                </p>
+                <p><span class="glyphicon glyphicon-book"></span> Published: 1993</p>
+                <hr>
+                <img class="img-responsive" src="http://placehold.it/900x300" alt="">
+                <hr>
+                <p> 
+                  “The Rock cries out to us today, <br>
+                  You may stand upon me, <br>
+                  But do not hide your face.”</p>
+                <a class="btn btn-primary" href="#">Read More <span class="glyphicon glyphicon-chevron-right"></span></a>
+                <a class="btn btn-primary" href="/media/Maya-Angelou-On-the-Pulse-of-the-Morning.mp3">Listen <span class="glyphicon glyphicon-music"></span></a>
```

Comment:
Nice work adding in the glyphicon-music. I think little touches like those go a long way towards the usability of the site.